.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
+max-line-length = 88
+select = C,E,F,W,B,B950
```

Comment:
why not leave the defaults?

Reply:
I agree with the [https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length]("don't go crazy if you can't fit into precisely 88 characters"-premise) and hence use the suggested settings there.

Reply:
the tl;dr is “it’s like highway speed limits, we won’t bother you if you overdo it by a few km/h” haha